src/Tests/Unit/Scrapers/Pipeline/Mediator/MetadataExtractors.test.ts

```
@@ -0,0 +1,264 @@
+ */
+function makeJsdomCtx(html: string, isVisible = true): Page | Frame {
+  const dom = new JSDOM(`<!DOCTYPE html><html><body>${html}</body></html>`);
+  const domDoc = dom.window.document;
+  return {
+    /**
+     * Execute callback with JSDOM document injected as global.
+     * @param fn - Evaluate callback.
+     * @param arg - Callback argument.
+     * @returns Resolved callback result.
+     */
+    evaluate: <T>(fn: (arg: unknown) => T, arg: unknown): Promise<T> => {
```

ESLint

[error] 'arg' is defined but never used.

(no-unused-vars)